SAMPLE_ENV_FILE

```diff
@@ -0,0 +1,11 @@
+create a .env file with the variables below to explore the app in dev mode
+
```

Comment:
Yes

Reply:
Then can we just remove it?

Reply:
If it is removed, anybody that forks this repo won't have any idea how to set up the environment variables because the original `.env` file is not meant to be pushed to git. Also, these variables may need to be set up in the `qa` server as well.